packages/contracts/src.ts/index.ts

```diff
@@ -347,7 +347,8 @@ function addContractWait(contract: Contract, tx: TransactionResponse) {
                 // Useful operations
                 event.removeListener = () => { return contract.provider; }
                 event.getBlock = () => {
```

Comment:
Not for now, but at some point we will need to remove the `getBlock` method from here